apps/client/.storybook/preview.js

```diff
@@ -1,4 +1,4 @@
-import '../assets/styles.css'
+import '..//styles.css'
```

Comment:
Accidental typo? `../styles.css`

(though interestingly through this i discovered that the number of `/` doesn't matter! output remains the same)

Reply:
Ack! Wild that it still worked 😅 Let me fix that